lib/payment_processor/go_cardless/webhook_handler/event_store.rb

```diff
@@ -0,0 +1,25 @@
+module PaymentProcessor
+  module GoCardless
+    module WebhookHandler
+      class EventStore
+
+        class << self
+          def event_exists?(event)
```

Comment:
a bit weird that a `?` method actually goes ahead and creates the record then returns false... maybe call this method `find_or_create_event`?